src/hooks/useRCAuth4Google.js

```diff
@@ -0,0 +1,74 @@
+import { useState, useContext } from 'react';
+import { useToastBarDispatch } from '@rocket.chat/fuselage-toastbar';
+import RCContext from '../context/RCInstance';
+import { useGoogleLogin } from './useGoogleLogin';
+import { useToastStore, useUserStore, useTotpStore } from '../store';
+
+export const useRCAuth4Google = () => {
+  const [userOrEmail, setUserOrEmail] = useState(null);
+
+  const { signIn, signOut } = useGoogleLogin(
+    process.env.REACT_APP_GOOGLE_CLIENT_ID
```

Comment:
there is no concept of envs in components, we should get this from the props

Reply:
yes also now we can remove GOOGLE_CLIENT_ID prop from chat input because I extracted login code in other file